.github/scripts/docathon-label-sync.py

```diff
@@ -0,0 +1,37 @@
+import os
+from github import Github
+import sys
+import re
+
+token = os.environ.get('GITHUB_TOKEN')
+
+repo_owner = "pytorch"
+repo_name = "tutorials"
+pull_request_number = int(sys.argv[1])
+
+g = Github(token)
+repo = g.get_repo(f'{repo_owner}/{repo_name}')
+pull_request = repo.get_pull(pull_request_number)
+pull_request_body = pull_request.body
+
+# get issue number from the PR body
+if re.search(r'#\d{1,5}', pull_request_body):
+    issue_number = int(re.findall(r'#(\d{1,5})', pull_request_body)[0])
```

Comment:
This is too brittle.
What if there is a text like "This is a follow-up of for my PR #12345", then `repo.get_issue` will fail.
Also I'm not sure `pull_request_body` returns "#12345" and not "https://github.com/pytorch/tutorials/issue/12345"

Reply:
As discussed separately, for this purpose, this solution will suffice.